plugins/vueMarkdown.ts

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue'
+import VueMarkdown from 'vue-markdown'
+Vue.use(VueMarkdown)
```

Comment:
```ts
import VueMarkdown from 'vue-markdown'

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VueMarkdown, {})
})
```
have you tried this?

Reply:
cc @Jarsen136

Reply:
Oh, I just tried this but it does not work, unfortunately. So let's keep the current implement.